web/templates/user_group_list_item.hbs

```diff
@@ -8,13 +8,15 @@
     </td>
     {{#if can_remove_members }}
     <td class="remove_member">
-        {{#if is_direct_member}}
-            {{> components/action_button label=(t "Remove") custom_classes="remove-member-button" attention="quiet" intent="danger" aria-label=(t "Remove") }}
-        {{else}}
-            <span class="tippy-zulip-tooltip" data-tippy-content="{{#if is_me}}{{t 'You are a member of {name} because you are a member of a subgroup ({subgroups_name}).'}} {{else}}{{t 'This user is a member of {name} because they are a member of a subgroup ({subgroups_name}).'}}{{/if}}">
-                {{> components/action_button label=(t "Remove") custom_classes="remove-member-button" attention="quiet" intent="danger" aria-label=(t "Remove") disabled="disabled" }}
-            </span>
-        {{/if}}
+        <div class="remove-button-wrapper">
```

Comment:
It's not obvious to me that we need this div -- couldn't the class just go on the containing `td`?

Probably not important though, merging since the transition is. But if you see that as an easy simplification, feel free to open a follow-up PR.